Backend/BackendAPI/Controllers/ValuesController.cs

```diff
@@ -29,7 +29,7 @@ public ActionResult<IEnumerable<string>> Get()
         public ActionResult<string> Get(int id)
         {
             if (id > 10)
-                throw new HttpResponseException("Za duze id", 409);
```

Comment:
Nawet nie wiedziałem że mieliśmy multilanguage xd

Reply:
To jest jakiś śmieciowy kontroler xd